charts/tfy-karpenter-config/values.yaml

```
@@ -37,6 +37,8 @@ karpenter:
     - alias: al2023@latest
     ## @param karpenter.defaultNodeTemplate.userData User data to append in the default node template. Set this to "" to disable injection.
     userData: ""
```

```suggestion
    ## @param karpenter.defaultNodeTemplate.userData User data to append in the default node template. Set this to "default" to let the chart automatically decide this. Set this to "" to disable this injection.
    userData: "default"
```

Should we unify this with `gpuDefaultNodeTemplate`  to make `userData` default as `"default"` ?

`""` can be used for explicitly clearing `userData` if any

---

For `defaultNodeTemplate` we don't need this

---

Sure, but this and gpu values being different creates inconsistency
Not blocking on this for now